.changeset/khaki-brooms-clean.md

```diff
@@ -0,0 +1,5 @@
+---
+"@localyze-pluto/components": major
+---
+
+Remove the `AlertDialog` component and replace it with the `ConfirmationModal` component which re-uses the `Modal` component to unify the appearance of modals
```

Comment:
Are we using the `AlertDialog`  somewhere? Shouldn't we deprecate the component first and them remove it in the next version?

Reply:
We have a major version bump so when we upgrade to the next version we can replace the `AlertDialog` with the `ConfirmationModal`.